docs/service-catalog/docs/040-cr-service-binding-usage.md

```diff
@@ -58,3 +58,21 @@ This table lists all the possible parameters of a given resource together with t
 | **status.conditions.type** |    **NO**   | Defines the type of the condition. The value of this field is always `Ready`. |
 | **message** |    **NO**   | Describes in a human-readable way why the ServiceBinding injection has failed. |
 | **reason** |    **NO**   | Specifies a unique, one-word, CamelCase reason for the condition's last transition. |
+
+
+## Related resources and components
+
+These are the resources related to this CR:
+
+| Custom resource   |   Description |
+|:----------:|:------|
+| UsageKind |  Provides information where to inject Secrets. |
+| [ServiceBinding](https://kubernetes.io/docs/concepts/extend-kubernetes/service-catalog/#api-resources) |  Provides Secrets to inject.  |
+
+
+These components use this CR:
+
+| Component   |   Description |
+|:----------:|:------|
+| Binding Usage Controller |  Reacts to every action of creating, updating or deleting ServiceBindingUsages in all Namespaces and uses ServiceBindingUsage data to inject binding. |
```

Comment:
- creating, updating or deleting -> `creating, updating, or deleting`
- Namespaces and uses -> `Namespaces. It uses` or `Namespaces, and uses`